syscall/hostcalls.c

```diff
@@ -1254,6 +1258,18 @@ oe_result_t _oe_syscall_nanosleep_ocall(
 }
 OE_WEAK_ALIAS(_oe_syscall_nanosleep_ocall, oe_syscall_nanosleep_ocall);
 
+oe_result_t _oe_syscall_clock_gettime_ocall(
+    int* _retval,
+    int clock_id,
+    struct oe_timespec* cur_time)
+{
+    OE_UNUSED(_retval);
+    OE_UNUSED(clock_id);
+    OE_UNUSED(cur_time);
+    return OE_UNSUPPORTED;
```

Comment:
Please update the `edl_opt_out` test accordingly to ensure this default implementation is covered.